cli/agent.go

```diff
@@ -40,6 +42,12 @@ func (c *AgentCommand) Execute(_ []string) error {
 		return errors.Wrap(err, "problem setting up logger")
 	}
 
+	wd, err := os.Getwd()
+	if err != nil {
+		return errors.Wrap(err, "problem getting working directory")
+	}
+	opts.WorkingDirectory = wd
```

Comment:
opts is not a pointer, so this won't set the option in the agent's copy, just in ours.

Reply:
reordered to avoid this bug.